Scripts/Helpers/ProxyNode.cs

```diff
@@ -614,6 +696,46 @@ public void ClearFeedbackRequests(IRequestFeedback caller)
                 RemoveFeedbackRequest(request);
             }
         }
+
```

Comment:
Just a quick callout to what data you are storing here and why would be helpful for curious users.
This isn't public API, but it does save data so we should at least do a cursory explanation.